lms/tasks/email_digests.py

```diff
@@ -108,6 +107,7 @@ def send_instructor_email_digest_tasks():
                     {
                         "h_userid": h_userid,
                         "created_before": created_before.isoformat(),
+                        "created_after": created_after.isoformat(),
```

Comment:
So `send_instructor_email_digest_tasks()` is once again calculating `created_after` (as 24hrs before `created_before`) and passing into `send_instructor_email_digest()`